REPs/REP-33.md

```diff
@@ -1,10 +1,10 @@
 ```
 REP: REP-33
 Title: Node State Transition
-Status: Draft
```

Comment:
You can't jump to final, see https://github.com/RSS3-Network/REPs/blob/main/REPs/REP-1.md#rep-workflow